src/coreclr/src/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -617,6 +617,14 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
             }
             break;
 
+            case NI_Vector64_ToVector128:
+            case NI_Vector64_ToVector128Unsafe:
+                if (op1Reg != targetReg)
```

Comment:
I couldn't use table driven logic for this one because `emitIns_R_R()` for `mov` expects `opt == NONE`.